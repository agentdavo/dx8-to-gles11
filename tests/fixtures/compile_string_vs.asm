#include "cs_vs/a.inc"
vs.1.1
#include "cs_vs/b.inc"
