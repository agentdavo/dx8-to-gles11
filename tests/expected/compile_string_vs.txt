VERTEX_ATTRIB idx=0
MULTITEXCOORD4F stage=0
